test/integration/main_test.go

```diff
@@ -97,6 +97,11 @@ func setMaxParallelism() {
 	// Each "minikube start" consumes up to 2 cores, though the average usage is somewhat lower
 	limit := int(math.Floor(float64(maxp) / 1.75))
 
+	// Windows tests were failing from timeouts due to too much parallelism
+	if runtime.GOOS == "windows" {
+		limit = int(limit / 2)
```

Comment:
`limit /= 2` should be enough here and should fix lint

Reply:
Done, thanks